src/models/set.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface Set {
```

Comment:
I think we should rename this to `ExerciseSet` because`Set` is a JS class (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)